src/home_robot/home_robot/utils/image.py

```diff
@@ -141,3 +143,8 @@ def smooth_mask(mask, kernel=None):
     # mask2 = cv2.dilate(mask2, kernel, iterations=1)
     mask2 = np.bitwise_and(mask, mask2)
     return mask, mask2
+
+
+def rotate_image(imgs: List[np.ndarray]) -> List[np.ndarray]:
```

Comment:
comment description here - this is a specific rotate to make stretch camera images up and down